testsuites/syncgateway/functional/tests/test_log_redaction.py

```diff
@@ -370,6 +370,73 @@ def test_sgCollectRestApi_errorMessages(params_from_base_test_setup, remove_tmp_
     assert "no such file or directory:" in resp.content.decode('ascii')
 
 
+@pytest.mark.syncgateway
+@pytest.mark.logredaction
+@pytest.mark.logging
+def test_log_content_verification(params_from_base_test_setup, remove_tmp_sg_redaction_logs):
+    """
+    @summary
+    1. Verify all the log files on sgcollect zip file exist after unzip
+    2. Verify content of sync_gateway.log
+    """
+
+    cluster_config = params_from_base_test_setup["cluster_config"]
+    mode = params_from_base_test_setup["mode"]
+    cluster_helper = ClusterKeywords(cluster_config)
+    cluster_hosts = cluster_helper.get_cluster_topology(cluster_config)
+    sg_admin_url = cluster_hosts["sync_gateways"][0]["admin"]
+    sg_url = cluster_hosts["sync_gateways"][0]["public"]
+    sg_platform = params_from_base_test_setup["sg_platform"]
+    sg_ip = host_for_url(sg_admin_url)
+    sg_conf_name = "sync_gateway_default"
+    sg_db = "db"
+    num_of_docs = 10
+    # cbs_ce_version = params_from_base_test_setup["cbs_ce"]
+    if get_sync_gateway_version(sg_ip)[0] < "2.1":
+        pytest.skip("log redaction feature not available for version < 2.1 ")
+
+    sg_conf = sync_gateway_config_path_for_mode(sg_conf_name, mode)
+
+    cluster = Cluster(config=cluster_config)
+    cluster.reset(sg_config_path=sg_conf)
+
+    # Get sync_gateway host
+    cluster = load_cluster_config_json(cluster_config)
+    sg_host = cluster["sync_gateways"][0]["ip"]
+    if cluster["environment"]["ipv6_enabled"]:
+        sg_host = "[{}]".format(sg_host)
+
+    # 3. Create user in sync_gateway
+    sg_client = MobileRestClient()
+    channels = ["logging"]
+    sg_client.create_user(url=sg_admin_url, db=sg_db, name='autotest', password='password', channels=channels)
+    autouser_session = sg_client.create_session(url=sg_admin_url, db=sg_db, name='autotest')
+
+    # 4. Create docs with xattrs
+    sgdoc_bodies = document.create_docs(doc_id_prefix='sg_docs', number=num_of_docs,
+                                        attachments_generator=attachment.generate_2_png_10_10, channels=channels)
+    sg_client.add_bulk_docs(url=sg_url, db=sg_db, docs=sgdoc_bodies, auth=autouser_session)
+    assert len(sgdoc_bodies) == num_of_docs
+
+    resp = sg_client.sgCollect_info(sg_host)
+    if resp["status"] != "started":
+        assert False, "sg collect did not started"
```

Comment:
assert resp["status"] != "started",  "sg collect did not started"

Reply:
done